src/Microsoft.TemplateEngine.Edge/Installers/NuGet/INuGetLogger.cs

```diff
@@ -0,0 +1,9 @@
+using Microsoft.TemplateEngine.Abstractions;
+using NuGet.Common;
+
+namespace Microsoft.TemplateEngine.Edge.Installers.NuGet
+{
+    public interface INuGetLogger : IIdentifiedComponent, ILogger
```

Comment:
I'm not fan of this interface...
It seems to me like we need general logging solution for .Edge, and `Microsoft.TemplateEngine.Edge.Installers.NuGet` would make implementation for `NuGet.ILogger` based on that logging...